lib/suggestion-list.coffee

```diff
@@ -76,14 +64,16 @@ class SuggestionList
   onDidConfirm: (fn) ->
     @emitter.on('did-confirm', fn)
 
-  selectNext: =>
+  selectNext: (e) =>
     @emitter.emit('did-select-next')
+    e.stopImmediatePropagation()
```

Comment:
Yes, it seems okay to me. There's really no other place to bind the event handler. Would prefer if you spelled out `event` though. Another thing I'm curious about is whether you need to add and remove these bindings programmatically... another approach would be to only perform the action and stop propagation if the autocomplete is already visible. That way you're not constantly adding and removing handlers.